src/components/ResponsiveFrame.js

```diff
@@ -8,14 +9,20 @@ import MinimapXYFrame from "./MinimapXYFrame";
 
 const createResponsiveFrame = Frame =>
   class ResponsiveFrame extends React.Component {
+    static propTypes = {
+      size : PropTypes.array
+    };
+
+    static defaultProps = {
+      size : [500, 500]
```

Comment:
There's code throughout the codebase that duplicates checking for an undefined props and then assigned some literal. All of those can be replaced with defaultProps to make future changes easier.

Reply:
Oh thanks for showing me this pattern like this. I'd seen it before and never really got it and just generally avoided statics because of it.